dev_guide/deployments/deployment_strategies.adoc

```diff
@@ -73,16 +73,20 @@ The following is an example of the Rolling strategy:
 strategy:
   type: Rolling
   rollingParams:
-    timeoutSeconds: 120 <1>
-    maxSurge: "20%" <2>
-    maxUnavailable: "10%" <3>
-    pre: {} <4>
+    updatePeriodSeconds: 1 <1>
+    intervalSeconds: 1 <2>
+    timeoutSeconds: 120 <3>
+    maxSurge: "20%" <4>
+    maxUnavailable: "10%" <5>
+    pre: {} <6>
     post: {}
 ----
-<1> How long to wait for a scaling event before giving up. Optional; the default is 120.
-<2> `maxSurge` is optional and defaults to `25%` if not specified; see below.
-<3> `maxUnavailable` is optional and defaults to `25%` if not specified; see below.
-<4> `*pre*` and `*post*` are both xref:lifecycle-hooks[lifecycle hooks].
+<1> The time to wait between individual pod updates.
+<2> The time to wait between polling deployment status after update.
+<3> How long to wait for a scaling event before giving up. Optional; the default is 120.
```

Comment:
> giving up means automatically rolling back to the previous complete deployment.

This is fine for now. Can you follow up with replacing the instance of "deployment" in our docs with "rollout" or "replication controller"? "rollout" when referring to the process of deploying, "replication controller" when talking about a version of the deployment config.